Sources/ProjectDescription/RunActionOptions.swift

```diff
@@ -128,3 +140,16 @@ extension RunActionOptions {
         }
     }
 }
+
+extension RunActionOptions {
+    public enum GPUFrameCaptureMode: String, Codable, Equatable {
+        case autoEnabled = "0"
+        case metal = "1"
+        case openGL = "2"
+        case disabled = "3"
```

Comment:
There isn't a need to declare the underlying Xcode values here ... this is handled by the XcodeProj library